### PR TITLE
7 post/api/reviews/review id/comments

### PR DESCRIPTION
**Pull request by JakeMaciver**

### CODE REVIEW

File: __tests__/endpoint.test.js [hunk 1]
```diff
@@ -202,4 +202,60 @@ describe('GET /api/reviews/:review_id/comments', () => {
...
+
+describe('POST /api/reviews/:review_id/comments', () => {
+  test('should return with status code 201', () => {
+		const commentToPost = { username: 'bainesface', body: 'Game was great' };
+		return request(app)
+			.post('/api/reviews/3/comments')
+			.send(commentToPost)
+			.expect(201);
+	});
+	test('200: return an object', () => {
```
Comment (HeatherEver) [inline __tests__/endpoint.test.js]: should be a 201 not 200

File: __tests__/endpoint.test.js [hunk 2]
```diff
@@ -202,4 +202,60 @@ describe('GET /api/reviews/:review_id/comments', () => {
...
+		return request(app)
+			.post('/api/reviews/2/comments')
+			.then(({ body }) => {
+				expect(typeof body).toBe('object');
+				expect(Array.isArray(body)).toBe(false);
+			});
+	});
+	test('200: should return with the comment that was posted to the database', () => {
```
Comment (HeatherEver) [inline __tests__/endpoint.test.js]: same here

File: __tests__/endpoint.test.js [hunk 3]
```diff
@@ -202,4 +202,60 @@ describe('GET /api/reviews/:review_id/comments', () => {
...
+  test('400: should return a 400 Bad request error when invalid user input', () => {
+    const commentToPost = {username: '', body: 'Game was great' };
+    return request(app)
+			.post('/api/reviews/3/comments')
+			.send(commentToPost)
+			.expect(400)
+			.then(({ body }) => {
+				expect(body).toEqual({ message: 'Bad request' });
+			});
+  });
```
Comment (HeatherEver) [inline __tests__/endpoint.test.js:260]: can you please add some more error handling for: 
```
Status 404, username does not exist
```

**Review by HeatherEver — APPROVED** [review]
lovely work here, just one more test to add :)

**Review by pippaamy — APPROVED** [review]
Great work, go ahead and merge! :)

**Pull request merged**